### ISSUE TITLE
Selected AutoDrive setting not remembered after restart game

### ISSUE THREAD

**Issue by JustDen1234** [Courseplay/courseplay#7394]
After restarting the game, the previous selected option for AutoDrive is reset to 'Don't use'.

![image](https://user-images.githubusercontent.com/68871402/128000394-25767eea-07a0-46cd-8dd2-56063c4aa842.png)

CP version : 6.4.0.5
AD version: 1.1.1.4

**Comment by Bobster82** [comment]
I'm using 6.4.0.4 (not the release version) I dont know exactly what commit Im at, but I dont have that issue

**Comment by JustDen1234** [comment]
It looks like it occurs only if you select the 'refill, unload and park' (not really sure it's called this way, not running the game at this time) option.

**Comment by Bobster82** [comment]
> It looks like it occurs only if you select the 'refill, unload and park' (not really sure it's called this way, not running the game at this time) option.

Yup ur right...
Its reset at restart when option refill/unload and park or just park is enabled.

**Comment by Bobster82** [comment]
I dived a bit into the code to help out CP team finding it (I think ;) ) I did my best, but I have a hard time finding my way into CP code (too big and I'm not experienced enough)

I added a log line to check what happend. (I add a screenshot with code, so u know what I mean) When the game loads, #self.values = 2 instead of 4, so it doesn't show the park options. When I update the hud (clicking on a hud item for example, the value is 4 again and the autodrive setting is as saved)

I hope it helped, else I need to learn more. haha

file: settings.lua
![image](https://user-images.githubusercontent.com/47780758/128162298-7e56f348-b59a-472e-acd1-4d58a1b8f322.png)
